roles/check_system_static/tasks/main.yml

```diff
@@ -16,8 +16,8 @@
     msg: "System versions lower than Red Hat Enterprise Linux / CentOS 7.3 have been deprecated. Please use CentOS 7.3 and above. See https://github.com/pingcap/docs/blob/master/op-guide/recommendation.md"
   when:
     - ansible_os_family == 'RedHat'
-    - ansible_distribution != 'Amazon'
-    - ansible_distribution_version < '7.3'
+    - ansible_distribution == 'CentOS' or ansible_distribution == 'RedHat'
```

Comment:
ansible_distribution in ['RedHat', 'CentOS']

Reply:
Done